src/gui/org/deidentifier/arx/gui/view/impl/utility/ViewClassificationAttributes.java

```diff
@@ -233,9 +231,11 @@ private void update() {
             item.dispose();
         }
         
-        for (int i = 0; i < handle.getNumColumns(); i++) {
-            createTableItem(handle, i, features, model.getSelectedFeatures());
-            createTableItem(handle, i, classes, model.getSelectedClasses());
+        for (int col = 0; col < handle.getNumColumns(); col++) {
+            String attribute = handle.getAttributeName(col);
+            Image image = controller.getResources().getImage(model.getInputDefinition().getAttributeType(attribute));

```

Comment:
This needs to be changed. If output data is available, we should use the according DataDefinition. This means:

DataDefinition definition = model.getInputDefinition();
if (model.getOutputDefinition() != null) {
definition = model.getOutputDefinition();
}